torch/testing/_asserts.py

```diff
@@ -734,15 +749,17 @@ def assert_close(
     :meth:`~torch.Tensor.qscheme` and the result of :meth:`~torch.Tensor.dequantize` is close according to the
     definition above.
 
-    :attr:`actual` and :attr:`expected` can be :class:`~torch.Tensor`'s or any array-or-scalar-like of the same type,
-    from which :class:`torch.Tensor`'s can be constructed with :func:`torch.as_tensor`. In addition, :attr:`actual` and
-    :attr:`expected` can be :class:`~collections.abc.Sequence`'s or :class:`~collections.abc.Mapping`'s in which case
-    they are considered close if their structure matches and all their elements are considered close according to the
-    above definition.
+    :attr:`actual` and :attr:`expected` can be :class:`~torch.Tensor`'s or any tensor-or-scalar-like that share the a
+    common superclass (if :attr:`allow_subclasses` is ``True``), from which :class:`torch.Tensor`'s can be constructed
```

Comment:
I think this would be "any tensor-like or scalar-like as long as :attr:\`actual\` and :attr:\`expected\` are the same types or one is a (direct or indirect) subclass of the other."?